### PR TITLE
Bump luakit webkit

### PR DESCRIPTION
**Pull request by oz123**
Bug: https://bugs.gentoo.org/893714

### CODE REVIEW

File: www-client/luakit/luakit-2.3.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+			dev-lua/luassert[${LUA_USEDEP}]
+			dev-lua/luacheck[${LUA_USEDEP}]
+		')
+		x11-base/xorg-server[xvfb]
+	)
+"
+#PATCHES=(
+#	"${FILESDIR}"/${PN}-2.2.1-make.patch
+#)
+#
```
Comment (juippis) [inline www-client/luakit/luakit-2.3.3.ebuild:52]: Can't you just remove these lines if they're not used?

File: www-client/luakit/luakit-2.3.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	tc-export CC PKG_CONFIG
+}
+
+src_compile() {
+	emake \
+		PREFIX="${EPREFIX}/usr" \
+		USE_LUAJIT=$(usex lua_single_target_luajit 1 0) \
+		${PN} ${PN}.so
+
+	#use doc && emake doc
```
Comment (juippis) [inline www-client/luakit/luakit-2.3.3.ebuild:65]: You still have `doc` use flag present in the ebuild. It doesn't seem tobe doing anything anymore? Maybe the use flag needs to be removed?

File: www-client/luakit/luakit-2.3.3.ebuild (hunk 1 above)
Comment (oz123) [reply www-client/luakit/luakit-2.3.3.ebuild]: Removed.

File: www-client/luakit/luakit-2.3.3.ebuild (hunk 2 above)
Comment (oz123) [reply www-client/luakit/luakit-2.3.3.ebuild]: It's used to make the docs, I think I temporarily disable it.

**Pull request closed without merging**